src/browser/webapi/element/Html.zig

```diff
@@ -67,36 +67,36 @@ pub fn construct(page: *Page) !*Element {
 }
 
 pub const Type = union(enum) {
-    anchor: Anchor,
-    body: Body,
-    br: BR,
-    button: Button,
+    anchor: *Anchor,
```

Comment:
This change is fine. The goal was to remove an unnecessary allocation and keep everything close together, but with your change, it's still 1 allocation instead of 3 for these...and they're still grouped together, so it's still an improvement for these cases.